nixos/modules/programs/java.nix

```diff
@@ -40,12 +38,38 @@ in
         type = types.package;
       };
 
+      binfmt = mkOption {
+        type = types.bool;
+        default = false;
+        description = lib.mdDoc "Register Java program execution with binfmt";
+      };
+
```

Comment:
This could possibly just use `mkEnableOption (lib.mdDoc "Java program execution with binfmt")` or something. But it's minor and more of a preference thing

Reply:
Probably but also a lot of modules do enable options like this.

Reply:
Yeah it's not really a concern, just a quick note. `mkEnableOption` automatically makes a description of "Whether to enable {whatever you put in the function argument}", so keep that in mind when using it

Reply:
Ah, gotcha